src/main/kotlin/no/nav/familie/ef/sak/service/PersonopplysningerService.kt

```diff
@@ -23,18 +25,31 @@ class PersonopplysningerService(private val personService: PersonService,
     fun hentPersonopplysninger(ident: String): PersonopplysningerDto {
         return runBlocking {
             val egenAnsattDeferred = async { familieIntegrasjonerClient.egenAnsatt(ident) }
-            val søker = withContext(Dispatchers.Default) { personService.hentSøker(ident) }
+            val personMedRelasjoner = withContext(Dispatchers.Default) { personService.hentPersonMedRelasjoner(ident) }
 
-            val fullmakter = søker.fullmakt.filter { it.motpartsRolle == MotpartsRolle.FULLMEKTIG }
+            val fullmakter = personMedRelasjoner.søker.fullmakt.filter { it.motpartsRolle == MotpartsRolle.FULLMEKTIG }
 
             val identer = fullmakter.map { it.motpartsPersonident } +
-                          søker.sivilstand.mapNotNull { it.relatertVedSivilstand }.filterNot { it.endsWith("00000") }
-            val identNavn = hentNavn(identer)
-
-            personopplysningerMapper.tilPersonopplysninger(søker, ident, fullmakter, egenAnsattDeferred.await(), identNavn)
+                          personMedRelasjoner.søker.sivilstand.mapNotNull { it.relatertVedSivilstand }.filterNot { it.endsWith("00000") }
+            val identNavn = hentNavn(identer + andreForelderIdenter(personMedRelasjoner))
+            personopplysningerMapper.tilPersonopplysninger(
+                    personMedRelasjoner,
+                    ident,
+                    fullmakter,
+                    egenAnsattDeferred.await(),
+                    identNavn
+            )
         }
     }
 
+    private fun andreForelderIdenter(personMedRelasjoner: SøkerMedBarn): List<String> =
+            personMedRelasjoner.barn.values
+                    .flatMap { it.familierelasjoner }
+                    .filter { it.relatertPersonsIdent != personMedRelasjoner.søkerIdent }
+                    .filter { it.relatertPersonsRolle != Familierelasjonsrolle.BARN }
```

Comment:
Disse to kan kanskje slås sammen til én filter? `filter { it.relatertPersonsIdent != personMedRelasjoner.søkerIdent && it.relatertPersonsRolle != Familierelasjonsrolle.BARN }`? 😄